test/src/main/java/io/strimzi/test/TestUtils.java

```diff
@@ -133,7 +133,7 @@ public static long waitFor(String description, long pollIntervalMs, long timeout
                 result = ready.getAsBoolean();
             } catch (Exception e) {
                 exceptionMessage = e.getMessage();
-                if (++exceptionCount == 1) {
+                if (++exceptionCount == 1 && exceptionMessage != null) {
                     // Log the first exception as soon as it occurs
                     LOGGER.error("Exception waiting for {}, {}", description, exceptionMessage);
```

Comment:
Should this be really ERROR level? I noticed it yesterday and found it very confusing because this _ERROR_ seemed to be in many test cases without meaning anything.

Reply:
I think you mean something like this...:
```
2021-05-05 12:23:12 [ForkJoinPool-1-worker-11] ERROR [TestUtils:138] Exception waiting for Wait for Kafka: my-cluster-1594415598 will have desired state: Ready, null
```
but that will disappear after `exceptionMessage != null`.

Reply:
Ok, than maybe it is covered 🤷‍♂️